### PR TITLE
[CSRS-303] - upload attachments

### PR DESCRIPTION
**Pull request by skiselev1**
# Description

This PR includes the following proposed change(s):

- [CSRS-303] - upload attachments

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/csrs-portal/src/app/api/api/account.service.ts [hunk 1]
```diff
@@ -368,6 +368,7 @@ export class AccountService {
         let localVarCredential: string | undefined;
         // authentication (Bearer) required
         localVarCredential = this.configuration.lookupCredential('Bearer');
+        console.log('localVarCredential ' + localVarCredential);
```
Comment (pbolduc) [inline src/frontend/csrs-portal/src/app/api/api/account.service.ts:371]: please do not leave console log statements in the front end.

File: src/frontend/csrs-portal/src/app/app.module.ts [hunk 2]
```diff
@@ -88,7 +103,8 @@ export function HttpLoaderFactory(http: HttpClient): TranslateHttpLoader {
         }
       ),
       deps: [OidcSecurityService],
-      multi: false
+      multi: false,
```
Comment (pbolduc) [inline src/frontend/csrs-portal/src/app/app.module.ts:106]: trailing comma, surprised this is not caught by es-lint

**Review by pbolduc — CHANGES_REQUESTED** [review]

**Review by kkadaru — APPROVED** [review]

**Review by pbolduc — APPROVED** [review]

File: src/frontend/csrs-portal/src/app/api/api/account.service.ts (hunk 1 above)
Comment (skiselev1) [reply src/frontend/csrs-portal/src/app/api/api/account.service.ts]: deleted console.log line

File: src/frontend/csrs-portal/src/app/app.module.ts (hunk 2 above)
Comment (skiselev1) [reply src/frontend/csrs-portal/src/app/app.module.ts]: removed trailing comma

**Pull request merged**